playbooks/roles/buildbot-master/tasks/main.yml

```diff
@@ -0,0 +1,79 @@
+---
+#
+# edX Configuration
+#
+# github:     https://github.com/edx/configuration
+# wiki:       https://github.com/edx/configuration/wiki
+# code style: https://github.com/edx/configuration/wiki/Ansible-Coding-Conventions
+# license:    https://github.com/edx/configuration/blob/master/LICENSE.TXT
+#
+#
+#
+# Tasks for role buildbot-master
+# 
+# Overview:
+# 
+#
+# Dependencies:
+#   - common
+#
+# 
+# Example play:
+#
+#
+
+- name: create buildbot master group
```

Comment:
I've noticed some roles start task names uppercase and some start them lowercase. Does it matter?